pennylane/hf/tapering.py

```diff
@@ -543,19 +545,16 @@ def optimal_sector(qubit_op, generators, active_electrons):
 def transform_hf(generators, paulix_ops, paulix_sector, num_electrons, num_wires):
     r"""Transform a Hartree-Fock state with a Clifford operator and taper qubits.
 
-    The Hartree-Fock (HF) state for a molecule is transformed to a qubit observable under Jordan-Wigner (JW)
-    transform. To do this, each occupied orbital in the original HF state is assigned with a fermionic creation
-    operator and each unoccupied orbital is replaced with an identity operator. The JW transform is then applied to
-    map the resulting fermionic operator to a qubit basis.
-
-    This observable is tapered using the same Cliffords :math:`U` that are obtained from the symmetries of the
-    molecular Hamiltonian. A new, tapered Hartree-Fock state is built by placing all the qubits (wires) which are
-    acted on by a Pauli-X or Pauli-Y operator in the :math:`|1\rangle` state, while leaving the rest of them in
-    the :math:`|0\rangle` state.
+    The fermionic operators defining the molecule's Hartree-Fock (HF) state are first mapped onto a qubit operator
+    using the Jordan-Wigner encoding. This operator is then transformed using the Clifford operators :math:`U`
+    obtained from the :math:`\mathbb{Z}_2` symmetries of the molecular Hamiltonian resulting in a qubit operator
+    that acts non-trivially only on a subset of qubits. A new, tapered HF state is built on this reduced subset
+    of qubits by putting the qubits that are acted on off-diagonally by the operator in the :math:`|1\rangle`
```

Comment:
What does "putting the qubits that are acted on off-diagonally by the operator in the |1> state" mean? I find this confusing

Reply:
@ixfoduap I meant that we place all the qubits which are acted on by a `Pauli-X` or `Pauli-Y` operators in the excited state while keeping the rest (the ones acted by Pauli-Z and Identity) in the ground state.